applications/charter/P4_Apps_WG_charter.mdk

```diff
@@ -42,13 +52,14 @@ we propose the WG produce the following deliverables initially.
 - Specifications for common telemetry-report format and semantics
 - Test applications for the above
 
-The specifications will be published at [P4.org](https://p4lang.github.io/p4-spec/)
+The specifications will be published at
+[P4.org](https://p4lang.github.io/p4-spec/)
 and the sample codes and test cases will be available to download at
 [GitHub repo](https://github.com/p4lang/p4-applications).
 
 Once we confirm demands for other application areas in future, we could 
 then create sub-WGs each of which is geared toward its own specific types of 
 applications. We expect to see sub-WGs on traffic load balancing, connection 
 load balancing, congestion control, source routing, security, application 
-acceleration, etc.
+acceleration, and other emerging areas.
```

Comment:
How about leave this a bit high-level for now and not get into sub-WGs yet. Also, the word "other areas" implies that there will be one focus initially and everything else will be considered later. But we are not saying anywhere in the charter that we will focus on INT first. How about -

"The working group will initially focus on In-band Network Telemetry. The working group will solicit interest from participants in other application areas such as Connection Load Balancing, Traffic Load Balancing, Congestion Control, Source Routing, Security, Application Acceleration, etc. and will consider incorporating these into the working group charter subsequently"

Reply:
Sounds good. We mention the initial focus area in the previous paragraph. I'm taking the rest of the suggested text.